rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -575,6 +585,83 @@ def test_exec(config, ssh_con):
+            time.sleep(30)
+            if srv_restarted is False:
+                raise TestExecError("RGW service restart failed")
+            else:
+                log.info("RGW service restarted")
+
+            bucket_name = utils.gen_bucket_name_from_userid(
+                user_info["user_id"], rand_no=cc
+            )
+            log.info(f"creating bucket {bucket_name} with s3 user")
+            bucket = reusable.create_bucket(bucket_name, s3_rgw_conn, user_info)
+            log.info("enable bucket version using s3 user")
+            reusable.enable_versioning(
+                bucket, s3_rgw_conn, user_info, write_bucket_io_info
+            )
+            utils.exec_shell_cmd(f"fallocate -l 4k obj4k")
+            s3cmd = "/home/cephuser/venv/bin/s3cmd"
+            range_val = f"1..{config.objects_count}"
+            cmd = (
+                "for i in {"
+                + range_val
+                + "}; do "
+                + f"{s3cmd} put obj4k s3://{bucket_name}/object-$i; done"
+            )
+            rc = utils.exec_shell_cmd(cmd)
+            if rc is False:
+                raise AssertionError(
+                    f"Failed to upload current object to bucket {bucket_name}"
+                )
+
+            response = utils.exec_shell_cmd(cmd)
+            if response is False:
+                raise AssertionError(
+                    f"Failed to upload non-current object to bucket {bucket_name}"
+                )
+
+            resp = utils.exec_shell_cmd(f"{s3cmd} ls s3://{bucket_name} | wc -l")
+            if int(resp) != config.objects_count:
+                raise TestExecError(
+                    f"enable versioning on bucket {bucket_name} success but object count miss match"
+                )
+            resp = json.loads(
+                utils.exec_shell_cmd(
+                    f"radosgw-admin bucket stats --bucket {bucket_name}"
+                )
+            )
+            if resp["usage"]["rgw.main"]["num_objects"] != config.objects_count * 2:
+                raise TestExecError(
+                    f"enable versioning on bucket {bucket_name} success but object count miss match"
+                )
+            container_name = f"{bucket_name}-swift"
+            reusable.create_container_using_swift(container_name, rgw, subuser_info[-1])
+
+            log.info(f"enable versioning on bucket {bucket_name} with swift user")
+            container = swiftlib.resource_op(
+                {
+                    "obj": rgw,
+                    "resource": "put_container",
+                    "args": [
+                        bucket_name,
+                        {"X-Versions-Location": container_name},
+                    ],
+                }
+            )
+            log.info(f"container {container}")
+            if container:
+                raise TestExecError(
+                    f"enable versioning on bucket {bucket_name} with swift user should fail"
```

Comment:
@anrao19 , please change the conditional and exception message to failure scenario
"enable versioning on bucket {bucket_name} with swift user failed"

Reply:
done @hmaheswa